src/orquestra/sdk/_ray/_build_workflow.py

```diff
@@ -239,7 +259,10 @@ def _ray_remote(*inner_args, **inner_kwargs):
                     unpacked=unpacked,
                 )
 
-    named_remote = client.add_options(_ray_remote, **ray_options)
+    try:
+        named_remote = client.add_options(_ray_remote, **ray_options)
+    except exceptions.TaskWrappingError:
+        raise
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Does the underlying Ray function get called here? I didn't expect `.add_options()` to raise this exception.